W-17118609 doc updates for Nov GA jh (#55)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/managing-apis.adoc

* *APIs in the portal* 
+
Shows the APIs that have been added to your portal. Administrators can add only those APIs for which they have Exchange Viewer permission. For more information, see xref:assigning-administrator-permissions.adoc[]. 
+
All administrators are automatically assigned the Exchange Viewer permission for all APIs added to the portal. By default, the visibility of the APIs is set to Administrators. The portal administrator can change the visibility or remove APIs in the *APIs in the portal* tab. 
+

> Existing portals are limited 350 APIs. If you opt in to scale the portal, the maximum number of APIs is 1,000.

Reviewer: ```suggestion
Existing portals are limited to 350 APIs. To increase the maximum number of APIs to 1,000, opt in to scale the portal.
```

> Existing portals can have a maximum of 350 APIs in your portal. If you opt in to scale the portal, the maximum of APIs is 1,000.

Reviewer 2: ```suggestion
Existing portals are limited 350 APIs. If you opt in to scale the portal, the maximum number of APIs is 1,000.
```

//image::aeh-api-description.png[View API Description]

== Opt In to Increase the Number of APIs In the Portal

> Enable portal scalability to increase the maximum number of APIs available in your portal to 1,000. Administrators opt-in from the *API management* page. 

Reviewer 2: ```suggestion
Enable portal scalability to increase the maximum number of APIs available in your portal to 1,000. Administrators opt in from the *API management* page. 
```

Reviewer 2: I guess it's really lc (management)

Author: Yes, unfortunately the UI text is in sentence case. I’ve pushed for initial cap, but it’s not their priority to change.

Reviewer 2: I think this change (removing the dash) is applicable, though.

> Enable portal scalability to increase the maximum number of APIs available in your portal to 1,000. Administrators opt in from the *API management* page. 

Reviewer: ```suggestion
Administrators can enable portal scalability in the *API management* page to increase the maximum number of APIs available in your portal to 1,000. 
```

> Once this option is enabled, you can't revert to the portal version supporting the maximum of 350 APIs.

Reviewer 2: ```suggestion
After enabling this option, you can't revert to the portal version that supports up to 350 APIs.
```

> New portals are automatically opted in. 

Reviewer 2: ```suggestion
New portals automatically scale to 1,000 APIs. 
```

> . Click *Learn more* from the side bar in *API management*.

Reviewer 2: ```suggestion
. Click *Learn more* from the sidebar in *API management*.
```

. Select *I understand and want to enable scalability.* and click *Enable*.
+
You can add up to 1,000 APIs to your portal from Exchange. 

== View APIs in Exchange
You can view information about an API directly from Exchange to determine whether you want to add the API in your portal.

From the *APIs in Exchange* tab, select an API from the list and select *View in Exchange*.

> To filter APIs by categories and tags, select an option from the *Any category* or *Any tag* dropdown and click *Apply*.

Reviewer 2: You could make this a partial, since you reuse it a lot.

Author: Good idea. I'll review the content that could be made into partials after this release.
